client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -30,6 +30,7 @@ trait Client[F[_]] {
     *            response body afterward will result in an error.
     * @return The result of applying f to the response to req
     */
+  @deprecated("Use run(req).use(f)", "0.21.5")
```

Comment:
I question whether this message should indicate, "Don't use `fetch`, but, if you must, do `run(req).use(f)` per the helpful discussion in https://github.com/http4s/http4s/issues/3345#issuecomment-619598068 regarding what's wrong with `fetch`. 

Thoughts, @rossabaker and @satorg?

I admit that I don't know if "why is it deprecated" should be placed inside of the `@deprecated`'s message.

Reply:
Sorry, didn't get, what is your concern with the message? Do you think it should be more descriptive?

Reply:
I think the message is in line with what we usually do.

Reply:
Thanks for addressing my question, satorg and Ross.